mantidproject/mantid
# Update all external links across codebase

**Description of work.**

This work assessed all links pointing to www.mantidproject.org. Those pointing to wiki pages that should have their documentation inside the code base have been moved to rst pages. Others have their addresses updated to point to archive.mantidproject.org

The MantidHelpWindow code has also been simplified so that a page can be called with a relative page path and it is assumed to be relative to the base paths of either the Qt Help or HTML documentation

**To test:**

To check the help window:

* Build the `docs-qthelp` target
* Start workbench and check the help buttons on interfaces and under the Help menu


Fixes #31340
Fixes #35017





---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?
- Are the release notes saved in a separate file, using Issue or PR number for file name and in the correct location?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
